tests/test_util.py

```diff
@@ -12,11 +12,11 @@ class TestItem:
 
 def test_attr_prepare():
```

Comment:
test_remove_unknown_fields?

Reply:
Done in https://github.com/scrapinghub/autoextract-poet/pull/9/commits/28849a55ecdee69e262bfa43a72750decf5efaaa